src/impl/project/diff/diffutils.ts

```diff
@@ -23,11 +28,39 @@ const pairAuaraRegExp = new RegExp(
 const deleteNotSupported = ["RecordType"];
 const LWC_IGNORE_FILES = ["jsconfig.json", ".eslintrc.json"];
 
+const UNSPLITED_METADATA_EXTENSION = [
```

Comment:
Too many duplications? Can we utilze our metadata info file for these extensions?